telemetrylib/SQL.cpp

```diff
@@ -0,0 +1,128 @@
+//
+// Created by Mingcan Li on 2/3/23.
+//
+
+#include "DTI.h"
+#include <thread>
+#include <unistd.h>
+
+class SQL : public DTI {
+public:
+    SQL(QString tableToCreate) {
+        this->tableToCreate = tableToCreate;
+
+        restclient = new QNetworkAccessManager();
+
+        t = new std::thread(&SQL::checkConnection, this);
+        t->detach();
+    }
+
+    ~SQL() {
+        finish = true;
+        t->join();
+    }
+
+    void sendData(QByteArray bytes) override {
+        auto curr_msec = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count();
+
+        QUrl myurl;
+        myurl.setScheme("http");
+        myurl.setHost("host"); // TODO
+        myurl.setPort(3000); // TODO
```

Comment:
You can replace all of these placeholders with the actual IP and port. Also, please remove the TODOs (sorry)

Reply:
removed